transformations/color_transformation/color_names.txt

```diff
@@ -0,0 +1,147 @@
+AliceBlue
```

Comment:
I would like to see this structured as a JSON or XML file. So that it can be easy in the future to add other language support.